internal/procedures/server/start.go

```diff
@@ -0,0 +1,57 @@
+package server
+
+import (
+	"fmt"
+
+	"github.com/edulinq/autograder/internal/api"
+	"github.com/edulinq/autograder/internal/common"
+	"github.com/edulinq/autograder/internal/config"
+	"github.com/edulinq/autograder/internal/db"
+	"github.com/edulinq/autograder/internal/log"
+	"github.com/edulinq/autograder/internal/model"
+	pcourses "github.com/edulinq/autograder/internal/procedures/courses"
+	"github.com/edulinq/autograder/internal/util"
+)
+
+func Start() error {
+	defer api.StopServers()
+
+	log.Info("Autograder Version.", log.NewAttr("version", util.GetAutograderFullVersion()))
+
+	err := common.WriteAndHandlePidStatus()
+	if err != nil {
+		return err
+	}
+
+	db.MustOpen()
```

Comment:
Since this code has been moved to a more general location, it can no longer use Must functions.